.yarn/patches/@metamask-assets-controllers-patch-7c7d711c8c.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/dist/token-prices-service/codefi-v2.cjs b/dist/token-prices-service/codefi-v2.cjs
+index 34f7bcf4dea1b8d6a1ea45051be09059d9d35353..6aa82360e63727852cda1719f5e893508b764e75 100644
+--- a/dist/token-prices-service/codefi-v2.cjs
++++ b/dist/token-prices-service/codefi-v2.cjs
+@@ -98,6 +98,8 @@ exports.SUPPORTED_CURRENCIES = [
+     'mxn',
+     // Malaysian Ringgit
+     'myr',
++    // Monad
++    'mon',
+     // Nigerian Naira
+     'ngn',
+     // Norwegian Krone
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/38261, which was not included in 13.11.0, but was cherry-picked in 13.10.4.